src/style.scss

```diff
@@ -0,0 +1,133 @@
+html {
+  --buy--color: #00acdc;
+  --text--color--one: #060b35;
+  --text--color--two: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text--color--two);
+}
+
+body {
+  margin: 0;
+}
+
+&card {
```

Comment:
look how we should use scss nesting
https://sass-lang.com/documentation/style-rules/parent-selector/